packages/react-components/src/components/styles/ParticipantItem.styles.ts

```diff
@@ -28,6 +28,23 @@ export const menuButtonContainerStyle = {
   width: '1.5rem'
 };
 
+/**
+ * @private
+ */
+export const participantStateMaxWidth = '5rem';
+/**
+ * @private
+ */
+export const participantStateStringStyles: IStyle = {
+  maxWidth: participantStateMaxWidth,
+  overflow: 'hidden',
+  textOverflow: 'ellipsis',
```

Comment:
Anytime I've done this I've also needed to do `whiteSpace: nowrap` - can you post a screenshot with a long text version of Connecting (I'll try it out later also when I get a chance)

Reply:
You are right. Needed to add whitespace no wrap otherwise a space was causing text to shift to a new line.
Added a screenshot as well